dev/ci/user-overlays/06493-gares-API-remove-big-file.sh

```diff
@@ -0,0 +1,8 @@
+if [ "$TRAVIS_PULL_REQUEST" = "6493" ] || [ "$TRAVIS_BRANCH" = "API/remove-big-file" ]; then
```

Comment:
Should now use variables CI_PULL_REQUEST and CI_BRANCH.

Reply:
Should or must?

Reply:
Must (but technically travis will still work)